packages/lib/src/interactions/utils.ts

```diff
@@ -53,3 +53,15 @@ export function getRatioEndPoint(
     startPoint.y + (heightSign * width) / ratio
   );
 }
+
+export function getDefaultInteractions(
+  keepRatio?: boolean
+): Record<string, Interaction> {
+  return {
+    Pan: {},
+    Zoom: {},
+    XAxisZoom: { modifierKey: 'Alt', disabled: keepRatio },
+    YAxisZoom: { modifierKey: 'Shift', disabled: keepRatio },
+    SelectToZoom: { modifierKey: 'Control' },
+  };
+}
```

Comment:
I made `DEFAULT_INTERACTIONS` a function for two reasons:
- It can disable dynamically `XAxisZoom` and `YAxisZoom` depending on `keepRatio`
- It returns a new object at each call which avoid mutation by lodash's `merge` in `DefaultInteractions`